diff --git a/src/Engines/JavaScript.js b/src/Engines/JavaScript.js
index 3dac80265ee0ec3e153d9bb4807d7aaa22245d6f..5636e697570fa0fa4f803a17d48d934f1d85bae7 100644
--- a/src/Engines/JavaScript.js
+++ b/src/Engines/JavaScript.js
@@ -64,6 +64,10 @@ class JavaScript extends TemplateEngine {
       }
       return mod;
     }
+
+    if ("default" in mod) {
+      return this._getInstance(mod.default);
+    }
   }
 
   getInstanceFromInputPath(inputPath) {
